lib/gui/utils/byte-size/byte-size.js

```diff
@@ -26,6 +26,11 @@
 const angular = require('angular');
 const MODULE_NAME = 'Etcher.Utils.ByteSize';
 const ByteSize = angular.module(MODULE_NAME, []);
+
+/* eslint-disable lodash/prefer-lodash-method */
```

Comment:
Aye, makes sense. Happy to keep it like it is then, since it's going to go away in the near future anyways.